tests/framework/ar/ActiveRecordTestTrait.php

```diff
@@ -161,23 +161,6 @@ public function testHasAttribute()
         $this->assertFalse($customer->hasAttribute(42));
     }
 
-    public function testFindScalar()
-    {
-        /* @var $customerClass \yii\db\ActiveRecordInterface */
-        $customerClass = $this->getCustomerClass();
-
-        /* @var $this TestCase|ActiveRecordTestTrait */
-        // query scalar
-        $customerName = $customerClass::find()->where(['id' => 2])->scalar('name');
-        $this->assertEquals('user2', $customerName);
-        $customerName = $customerClass::find()->where(['status' => 2])->scalar('name');
-        $this->assertEquals('user3', $customerName);
-        $customerName = $customerClass::find()->where(['status' => 2])->scalar('noname');
-        $this->assertNull($customerName);
-        $customerId = $customerClass::find()->where(['status' => 2])->scalar('id');
-        $this->assertEquals(3, $customerId);
-    }
-
```

Comment:
according to @cebe, 'when the framework and extensions where in one repo that trait was shared between DB AR implementation and other, like Elasticsearch, redis and mongo. When we split the repos out, that was the workaround' more info https://github.com/yiisoft/yii2-redis/blob/master/tests/bootstrap.php#L17-L26